dll/shellext/zipfldr/zipfldr.rc

```diff
@@ -6,7 +6,7 @@
 164 BITMAP "res/zipfldr.bmp"
 200 BITMAP "res/header.bmp"
 201 BITMAP "res/watermark.bmp"
-1 ICON "res/zipfldr.ico"
+IDI_ZIPFLDR ICON "res/zipfldr.ico"
```

Comment:
I know it's out of scope, but while you are already editing this file you could also exchange the other numbers with their identifiers. This is just bonus, not necessary for approvement.

Reply:
Done.